_docs/howto/host-from-home.md

```diff
@@ -0,0 +1,27 @@
+---
+title:    Host from Home
+category: Get Started
+order:    4
+---
+
+Any site you [create](/docs/howto/create-a-site.html) or save will be hosted from your computer. To save: click the <span class="fa fa-floppy-o"></span> in the URL bar.
```

Comment:
Instead of using an icon here, what do you think of saying "click the save icon in the URL bar."? I think with the screenshot immediately following using an icon isn't necessary.